scripts/cybershake/queue_monitor.py

```diff
@@ -58,11 +59,43 @@ def get_queue_entry(
     )
 
 
+def sacct_metadata(db_running_task: SlurmTask, task_logger: Logger, root_folder: str):
+    cmd = "sacct -n -X -j {} -o 'jobid%10,jobname%35,Submit,Start,End,NCPUS,CPUTimeRAW%18,State,Nodelist%60'"
+    output = (
+        subprocess.check_output(cmd.format(db_running_task.job_id), shell=True)
+        .decode("utf-8")
+        .strip()
+        .split()
+    )
+    # ['578928', 'u-bl689.atmos_main.18621001T0000Z', '2019-08-16T13:05:06', '2019-08-16T13:12:56', '2019-08-16T14:58:28', '1840', '11650880', 'COMPLETED', 'nid00[166-171,180-196]']
+    submit_time, start_time, end_time = [x.replace("T", "_") for x in output[2:5]]
+    n_cores = float(output[5])
+    run_time = float(output[6]) / n_cores
+    sim_dir = sim_struct.get_sim_dir(root_folder, db_running_task.run_name)
+    log_file = os.path.join(sim_dir, "ch_log", const.METADATA_LOG_FILENAME)
+    with open("/home/melody.zhu/meta.txt", "a") as f:
```

Comment:
this looks like debug message as well? why not just use the logger? and why is it hard coded?

Reply:
Thanks! All deleted